docs/security/README.md

```diff
@@ -0,0 +1,227 @@
+# Minio Security Overview [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+## Server-Side Encryption
+
+Minio supports two different types of server-side encryption (SSE):
+ - **SSE-C**: The Minio server en/decrypts an object with a secret key provided by the S3 client
+              as part of the HTTP request headers. Therefore SSE-C requires TLS/HTTPS.
+ - **SSE-S3**: The Minio server en/decrypts an object with a secret key managed by a KMS.
+               Therefore Minio requires a valid KMS configuration for SSE-S3.
+
+### Server-Side Encryption - Preliminaries
+
+#### Secret Keys
+
+The Minio server uses an unique, randomly generated secret key per object a.k.a *OEK*. Neither the 
```

Comment:
Change aka to also know as, define OEK acronym: 
"also known as, Object Encryption Key (*OEK*)".